Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp

```diff
@@ -207,18 +232,45 @@ class ApplicationManifestParserTest : public testing::Test {
         EXPECT_EQ(expectedValue, value);
     }
 
+    void testCategories(const Vector<String>& expectedValue)
+    {
+        StringBuilder builder;
+        builder.append("[");
+        for (size_t i = 0; i < expectedValue.size(); i++) {
```

Comment:
I would use a modern for loop:
```
for (auto& value : expectedValue) {
    if (builder.length() > 1)
        builder.append(", "_s);
```

Reply:
Switched to a modern for loop. We don't want a trailing `,` for the last element (since JSON spec specifies the last element shouldn't have a trailing `,`), but need one between the first and second element, so I believe this should be:
`if (builder.length() < expectedValue.size())` instead.